### ISSUE TITLE
tv_grab_uk_tvguide does not work

### ISSUE THREAD

**Issue by qdacsvx** [XMLTV/xmltv#160]
XMLTV Version?
--------------
head from github.

XMLTV Component?
----------------
tv_grab_uk_tvguide

What happened?
--------------
I have a configured grabber but grabbing no longer works.

What did you expect to happen?
------------------------------
Running the grabber downloads schedules to XML file.

Did you see any warnings/errors?
--------------------------------
No channels can download.

time tv_grab_uk_tvguide --days 1 --nodetailspage --debug --output guide0.xml
Fetching https://my.tvguide.co.uk/channellistings.asp?ch=112&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=112&cTime=01/20/2022%2000:00:00 from server.
Unable to retrieve web page for 112 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=113&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=113&cTime=01/20/2022%2000:00:00 from server.
Unable to retrieve web page for 113 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=120&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=120&cTime=01/20/2022%2000:00:00 from server.
Unable to retrieve web page for 120 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=123&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=123&cTime=01/20/2022%2000:00:00 from server.
Unable to retrieve web page for 123 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=126&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=126&cTime=01/20/2022%2000:00:00 from server.
Unable to retrieve web page for 126 

...

...

time tv_grab_uk_tvguide --days 1 --nodetailspage --debug --output guide0.xml
Fetching https://my.tvguide.co.uk/channellistings.asp?ch=112&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=112&cTime=01/20/2022%2000:00:00 from cache without checking with server.
Unable to retrieve web page for 112 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=113&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=113&cTime=01/20/2022%2000:00:00 from cache without checking with server.
Unable to retrieve web page for 113 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=120&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=120&cTime=01/20/2022%2000:00:00 from cache without checking with server.
Unable to retrieve web page for 120 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=123&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=123&cTime=01/20/2022%2000:00:00 from cache without checking with server.
Unable to retrieve web page for 123 

Fetching https://my.tvguide.co.uk/channellistings.asp?ch=126&cTime=01%2F20%2F2022%2000%3A00%3A00 from server.
Fetching https://www.tvguide.co.uk?ch=126&cTime=01/20/2022%2000:00:00 from cache without checking with server.
Unable to retrieve web page for 126 
...

Any other information?
----------------------
(For example, is this a new or intermittent issue?)
New.

What other software are you using?
----------------------------------
**Operating System:** Xubuntu

**Perl Version:** Perl 5 Version 30

**Comment by honir** [comment]
Works for me on those channels. Looks like an internet issue. Delete your cache before trying again.

**Comment by qdacsvx** [comment]
Confirmed. I was using an older version of tv_grab_uk_tvguide to do this grab. I tried to install head but couldn't configure it due to the website problem. I tried on another computer which does have head installed (and previously configured). There is no problem running the head version of the grabber. All the channels are downloading as normal. Fix: upgrade XMLTV to the latest version. Packaged versions are affected?
